doc/build-recorder-schema.ttl

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2023 Fotios Valasiadis
+# Copyright (C) 2023 Alexios Zavras
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+@prefix b: <http://example.org/build-recorder#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Types
+b:process a rdfs:Class .
```

Comment:
`rdfs` namespace definition is missing

Reply:
Note to self: Don't open pull requests 5 am in the morning without proper sleep.